Sources/Danger/Plugins/SwiftLint/ShellExecutor.swift

```diff
@@ -8,16 +9,16 @@ public enum SpawnError: Error {
 }
 
 internal class ShellExecutor {
-    func execute(_ command: String, arguments: [String] = [], environmentVariables: [String] = []) -> String {
-        let script = [environmentVariables.joined(separator: " "),
-                      command,
+    func execute(_ command: String, arguments: [String] = [], environmentVariables: [String: String] = [:]) -> String {
+        let script = [command,
                       arguments.joined(separator: " ")].filter { !$0.isEmpty }.joined(separator: " ")
-        print("Executing \(script)")
+        print("Executing `\(script)` with environment variables \(environmentVariables)")
```

Comment:
@f-meloni this was here before, but I have updated it to include the environment variables now.

Should I remove this or keep it?

If I keep I'll update it to print the merged envs and also add it to the `spawn` function below?

Reply:
Yes, I like it :)